controller/internal/supervisor/iptablesctrl/iptables.go

```diff
@@ -817,3 +857,102 @@ func flushUDPConntrack(networks []string) {
 	// 	}
 	// }
 }
+
+// SetTargetNetworks updates ths target networks. There are three different
```

Comment:
Let's move all the instance related structures and methods in a different file. It is very confusing to have them in the same file. The corresponding unit tests must be also in a different file.

Reply:
Yes, that will make it more readable.